_literate/A-stacking.jl

```diff
@@ -334,11 +337,13 @@ estack = rms(yhat(), y())
 
 @from_network mach begin
     mutable struct MyTwoModelStack
-        regressor=judge
+        regressor1=model1
+        regressor2=model2
+        judge=judge
     end
 end
 
-MyTwoModelStack_inst = MyTwoModelStack()
+My_Two_Model_Stack = MyTwoModelStack()
 
```

Comment:
Nah. In julia the convention is CamelCase for types and snake case for instances. So `my_two_model_stack` please.

Reply:
Looks like I had that syntax already on my local version; must not have pushed the latest changes then.